sdk/servicebus/service-bus/test/stress-test-track-2/scenarioRenewSessionLock.ts

```diff
@@ -81,21 +84,27 @@ export async function scenarioRenewSessionLock() {
       elapsedTime < testDurationForSendInMs &&
       stressBase.messagesSent.length < totalNumberOfMessagesToSend
     ) {
-      await stressBase.sendMessages([sender], numberOfMessagesPerSend, true);
+      await stressBase.sendMessages(
+        [sender],
+        numberOfMessagesPerSend,
+        true,
+        false,
```

Comment:
Any chance you can store these in variables that describe what they do? Looking at this code, I have no idea what `true, false` is meant to do!

Reply:
Nice suggestion, will update!

Reply:
Updated